date/date_c.php

```diff
@@ -695,7 +695,7 @@ public static function listAbbreviations() {}
      * Returns a numerically indexed array with all timezone identifiers
      * @param int $timezoneGroup
      * @param string $countryCode
-     * @return array|false
+     * @return array
      * @link https://php.net/manual/en/datetimezone.listidentifiers.php
      */
     public static function listIdentifiers($timezoneGroup = DateTimeZone::ALL, $countryCode = null) {}
```

Comment:
According to https://php.net/manual/en/datetimezone.listidentifiers.php `false` could be returned for PHP < 8. Please use #LanguageLevelTypeAware attribute to set return type in this case

Reply:
Done. I edited the description to explain the return type for PHP8 as well.